packages/transport-webtransport/test/transport.spec.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-console */
+/* eslint-env mocha */
+
+import { createEd25519PeerId } from '@libp2p/peer-id-factory'
+import { multiaddr } from '@multiformats/multiaddr'
+import { expect } from 'aegir/chai'
+import { webTransport, type WebTransportComponents } from '../src/index.js'
+
+describe('WebTransport Transport', () => {
+  let components: WebTransportComponents
+
+  beforeEach(async () => {
+    components = {
+      peerId: await createEd25519PeerId()
+    }
+  })
+
+  it('transport filter filters out invalid multiaddrs', async () => {
+    const valid = [
+      multiaddr('/ip4/1.2.3.4/udp/1234/quic-v1/webtransport/certhash/uEiAUqV7kzvM1wI5DYDc1RbcekYVmXli_Qprlw3IkiEg6tQ/certhash/uEiAUqV7kzvM1wI5DYDc1RbcekYVmXli_Qprlw3IkiEg6tQ/p2p/12D3KooWGDMwwqrpcYKpKCgxuKT2NfqPqa94QnkoBBpqvCaiCzWd')
```

Comment:
Should we add a webtransport relay address here?

Reply:
The first entry in the `invalid` array is a webtransport relay address, unless I'm misunderstanding you.  Maybe suggest an edit?

Reply:
```suggestion
      multiaddr('/ip4/1.2.3.4/udp/1234/quic-v1/webtransport/certhash/uEiAUqV7kzvM1wI5DYDc1RbcekYVmXli_Qprlw3IkiEg6tQ/certhash/uEiAUqV7kzvM1wI5DYDc1RbcekYVmXli_Qprlw3IkiEg6tQ/p2p/12D3KooWGDMwwqrpcYKpKCgxuKT2NfqPqa94QnkoBBpqvCaiCzWd'),
      multiaddr('/ip4/167.235.53.235/udp/4001/quic-v1/webtransport/certhash/uEiDCKMoprQ0dsmyQ874-yN7h5Nwd1NVMFTeW5Rgg6RXD3Q/certhash/uEiBBN1_poJQS1J7pkbR7nDsFqFjRgrN51RuBlXsgwdA4Ag/p2p/12D3KooWPKRSU7mX3JiiasLBKYsFhPgiGZNcdv5SxBZrbrU7C8Jw/p2p-circuit/p2p/12D3KooWRG7hywMqYLDWh6EhzoYn5nUMJnHytjjuEK4UvKu2b1JQ')
```

or into invalid addresses if that's where it belongs

Reply:
Is this not the same sort of address as the first entry in the invalid list?